config/branch/eos4.0.ini

```diff
@@ -0,0 +1,8 @@
+# eos4.0 branch settings
+
+[ostree]
+# A different deploy ref is needed for eos3.4+ so that users are
+# required to get a specific eos-updater on the eos3.3 branch.
+#
+# https://phabricator.endlessm.com/T22056
+stable_branch = latest1
```

Comment:
Since eos4.0 builds are already running from the eos4.0 branch of this repo, I don't think we need to bother with configuration (even though it's nice documentation).